docs/docs/configuration/skipping.md

```diff
@@ -11,12 +11,19 @@ This section details ways to specify the files and directories that Trivy should
 |     License      |     ✓     |
 
 By default, Trivy traverses directories and searches for all necessary files for scanning.
-You can skip files that you don't maintain using the `--skip-files` flag.
+You can skip files that you don't maintain using the `--skip-files` flag, or the equivalent YAML config option.
```

Comment:
can we say "or using the equivalent Trivy YAML configuration." instead?
To make it clear that we don't mean Kubernetes YAML or similar